File: src/_02_StructuralPattern/_02_05_Facade/java/before/EmailLib/Transport.java
package _02_StructuralPattern._02_05_Facade.java.before.EmailLib;

public class Transport {

  public static void send(MimeMessage message) {
    System.out.println(message.getFrom().getAddress());
    System.out.println(message.getRecipients().stream().map(InternetAddress::getAddress));
    System.out.println(message.getSubject());
    System.out.println(message.getText());
  }
  
}

File: src/_02_StructuralPattern/_02_05_Facade/EmailLib/Transport.js
/**
 * @typedef { import("./MimeMessage").default } MimeMessage
 */

export default class Transport {
  /** @param { MimeMessage } message */
  static send(message) {
    console.log(message.getFrom().getAddress());
    console.log(message.getRecipients().map(r => r.getAddress()));
    console.log(message.getSubject());
    console.log(message.getText());
  }
}